meilisearch-http/src/helpers/env.rs

```diff
@@ -1,10 +1,11 @@
+use meilisearch_lib::milli::heed::Env;
```

Comment:
what about re-exporting `heed` directly, since this is the only thing we need from milli on `meilisearch-http` side
```suggestion
use meilisearch_lib::heed::Env;
```

Reply:
Yup! Done!